scripts/images/allow.yaml

```diff
@@ -1,3 +1,24 @@
 rke2-images-multus:
   - docker.io/rancher/hardened-multus-cni
   - docker.io/rancher/hardened-cni-plugins
+rancher:
+  - docker.io/rancher/rancher
+  - docker.io/rancher/system-agent
+  - docker.io/rancher/shell
+  - docker.io/rancher/rancher-webhook
+  - docker.io/rancher/gitjob
+  - docker.io/rancher/fleet
+  - docker.io/rancher/fleet-agent
+  - docker.io/rancher/system-upgrade-controller
+  - docker.io/rancher/kubectl
+  - docker.io/rancher/mirrored-directxman12-k8s-prometheus-adapter
+  - docker.io/rancher/mirrored-prometheus-node-exporter
+  - docker.io/rancher/mirrored-prometheus-operator-prometheus-config-reloader
+  - docker.io/rancher/mirrored-prometheus-operator-prometheus-operator
+  - docker.io/rancher/mirrored-prometheus-prometheus
+  - docker.io/rancher/mirrored-kiwigrid-k8s-sidecar
+  - docker.io/rancher/mirrored-library-busybox
+  - docker.io/rancher/mirrored-grafana-grafana
+  - docker.io/rancher/mirrored-library-nginx
+  - docker.io/rancher/mirrored-kube-state-metrics-kube-state-metrics
+  - docker.io/rancher/mirrored-jettech-kube-webhook-certgen
```

Comment:
can we add a `custom` category that allows users to specify customized images, e.g untagged version of a `master-head release?

Reply:
The list here acts as an allow-list to select images from an image list (it doesn't check tags).
If an image list contains the image `rancher/rancher:master-head`, it will be included because we have `docker.io/rancher/rancher`